plug-ins/prometheus/architecture.md

```diff
@@ -0,0 +1,80 @@
+# Prometheus Plugin Architecture
+
+This document describe the architecture for the Prometheus Plugin (eosio::prometheus_plugin).  The Prometheus Plugin instruments nodes in order to allow
+data to be collected from [Prometheus](https://prometheus.io/).  The requirements for this plugin are described in this [Github issue](https://github.com/eosnetworkfoundation/mandel/issues/67).
```

Comment:
Let's incorporate a copy of the linked issue text in this document rather than include it by reference.  It being in the `mandel` repository issues makes it seem like an awfully fragile reference to me.